project/plugins.sbt

```diff
@@ -1,9 +1,9 @@
 resolvers += Classpaths.sbtPluginReleases
 
+addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.2")
```

Comment:
Can we move onto a real code formatter akka Scalafmt? I'm happy to do a PR after this one if we agree one this.